### ISSUE TITLE
measurement data in `Blasi_CellSystems2016` mistakenly transformed?

### ISSUE THREAD

**Issue by fbergmann** [Benchmarking-Initiative/Benchmark-Models-PEtab#26]
It seems like the measurement data in `Blasi_CellSystems2016` was transformed unintentionally. Back in the Berlin hackathon the data looked like this: 

```
observableId	preequilibrationConditionId	simulationConditionId	measurement	time	observableParameters	noiseParameters	observableTransformation	noiseDistribution
0ac		control	0.747750754619803	inf		sigma	log	normal
0ac		control	0.748832498938945	inf		sigma	log	normal
0ac		control	0.746418618725156	inf		sigma	log	normal
0ac		control	0.746019737149623	inf		sigma	log	normal
0ac		control	0.734677745641262	inf		sigma	log	normal
0ac		control	0.747368344860373	inf		sigma	log	normal
```

now it looks like so

```
observableId	preequilibrationConditionId	simulationConditionId	measurement	time	observableParameters	noiseParameters
observable_0ac		control	2.1122437151133617	inf		sd_0ac
observable_0ac		control	2.1145298590010246	inf		sd_0ac
observable_0ac		control	2.1094317927780883	inf		sd_0ac
observable_0ac		control	2.1085905470393707	inf		sd_0ac
observable_0ac		control	2.084810045031652	inf		sd_0ac
observable_0ac		control	2.1114361270525337	inf		sd_0ac
```

while the original data did mention `observableTransformation`, the data should not have been transformed.

**Comment by dweindl** [comment]
> @elbaraim Do you know this?

@elbaraim : Do you? :)

**Comment by elbaraim** [comment]
I did not implement this model so I do not know about the original data collection, however as I remember I ran the conversion into linear space of all `log` measurements.
In this case `log` which translated in `exp(value)`.

**Comment by JanHasenauer** [comment]
I checked. The original file before updating was correct:

- [ ]  Data should not be transformed. The original data measurement values were correct.

- [ ]  'log' indicated that a log-normally distributed noise was assumed. This should also be included in the updated case.

- [ ] The sigma of all observations should be the same, meaning that there is the same relative measurement noise. In the current file there are different sigmas for different observables. This is incorrect and I'm not sure why and how this was introduced. If this was done by a script, all other files to which this script was applied should also be checked.

**Comment by LeonardSchmiester** [comment]
These issues should be fixed. The Simulations also look ok to me. Can you point me to a figure or something else that I can use to compare the results @JanHasenauer ? I didn't find anything in the publication so far.

**Comment by JanHasenauer** [comment]
Measurement data and simulation should be shown in Figure 2C of the original publication:
https://www.cell.com/fulltext/S2405-4712(16)00003-X
As I'm not sure that we have exactly the same parameters, simulation could also be slightly different.

**Comment by LeonardSchmiester** [comment]
Thanks. To me, the model and the simulations look good.

**Comment by LeonardSchmiester** [comment]
Fixed by #69
